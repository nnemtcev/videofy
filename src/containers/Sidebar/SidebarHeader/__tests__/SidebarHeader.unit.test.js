import { shallow } from "enzyme";
import React from "react";
import { SidebarHeader } from "../SidebarHeader.js";

describe("SidebarHeader", () => {
  test("SidebarHeader renders with title prop passed in", () => {
    const wrapper = shallow(<SidebarHeader title="subscriptions" />);
    expect(wrapper).toMatchSnapshot();
  });

  test("SidebarHeader renders with null title prop", () => {
    const wrapper = shallow(<SidebarHeader />);
    expect(wrapper).toMatchSnapshot();
  });

  test("SidebarHeader renders with empty title prop passed in", () => {
    const wrapper = shallow(<SidebarHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
