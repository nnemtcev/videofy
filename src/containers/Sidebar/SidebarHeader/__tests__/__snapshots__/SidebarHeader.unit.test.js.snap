// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarHeader SidebarHeader renders with empty title prop passed in 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SidebarHeader SidebarHeader renders with null title prop 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SidebarHeader SidebarHeader renders with title prop passed in 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    SUBSCRIPTIONS
  </MenuHeader>
</MenuItem>
`;
